sdk/tables/Azure.Data.Tables/src/Sas/TableAccountSasQueryParameters.cs

```diff
@@ -0,0 +1,246 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+
+namespace Azure.Data.Tables.Sas
+{
+    /// <summary>
+    /// A <see cref="TableAccountSasQueryParameters"/> object represents the components
+    /// that make up an Azure Storage Shared Access Signature's query
+    /// parameters.  You can construct a new instance using
+    /// <see cref="TableAccountSasBuilder"/>.
+    ///
+    /// For more information, <see href="https://docs.microsoft.com/rest/api/storageservices/create-service-sas">Create a service SAS</see>.
+    /// </summary>
+    public sealed class TableAccountSasQueryParameters //: SasQueryParameters
```

Comment:
I don't think we need the commented base class name (I think this was copied from `TableSasQueryParameters` where it was accidentally left in as well). But this brings up a good question - could we consolidate the duplicate properties across this and `TableSasQueryParameters` into a `SasQueryParameters` base class?

Reply:
If you can make `TableSasQueryParameters` derive from `TableAccountSasQueryParameters`, then that'll be okay.  But adding a base class that we don't expect customers to use or meaningfully polymorph over is generally frowned upon.